rust/platform/triple_mappings.bzl

```diff
@@ -276,5 +293,6 @@ def triple_to_constraint_set(target_triple):
     constraint_set += vendor_to_constraints(triple_struct.vendor)
     constraint_set += system_to_constraints(triple_struct.system)
     constraint_set += abi_to_constraints(triple_struct.abi)
+    constraint_set += extra_ios_constriants(triple_struct)
```

Comment:
Do you know if there's a downside to having duplicate constraints? The only thing that bothers me about the change is that it seems there's overlapping checks here that may one day result in an invalid configuration. Should `ios` hosts separately evaluate constraints? I.e.:
```python
if triple_struct.system == "ios":
    constraint_set = ios_constraints(triple_struct)
else:
    # what's already here.
```

Reply:
I don't think there's any downside for duplications, but I haven't verified. But in this current setup there shouldn't be any duplicates since this new function doesn't add the CPU / system ones, and no other functions add the simulator / device ones (anymore)